pkg/api/deep_copy_generated.go

```diff
@@ -1004,12 +1004,26 @@ func deepCopy_api_DockerBuildStrategy(in buildapi.DockerBuildStrategy, out *buil
 	} else {
 		out.PullSecret = nil
 	}
+	if in.Env != nil {
```

Comment:
can you reverse this if? So you just set Env to nil and return and nuke the `else`